arch/sim/src/sim/up_unblocktask.c

```diff
@@ -129,7 +133,11 @@ void up_unblock_task(FAR struct tcb_s *tcb)
 
           /* Then switch contexts */
 
+#ifdef CONFIG_SIM_PREEMPTIBLE
+          swapcontext(&prev_rtcb->xcp.context, &rtcb->xcp.context);
```

Comment:
As I said before, you shouldn't call host function directly inside NuttX code.

Reply:
I removed this call